lib/BUILD

```diff
@@ -0,0 +1,6 @@
+cc_library(
+    name = "hello_world_lib",
+    srcs = glob(["**/*.cc"]),
```

Comment:
Not sure if we want to list only the required .cc files e.g. `hello_world.cc`. SImilar case with the headers

Reply:
Good point. That way we will be able to make separate libraries and import only what we need instead of importing all the files at once.